### ISSUE TITLE
Line Edits don't select entire text on double click

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange-widget-base#165]
##### Orange version
3.30.dev

##### Expected behavior
Line Edit selects entire text on double click.

##### Actual behavior
Only a part of the text is selected.

##### Steps to reproduce the behavior
Double click a longer word in Line Edit (i.e. Edit Domain with heart disease data).

##### Additional info (worksheets, data, screenshots, ...)
![edit](https://user-images.githubusercontent.com/12524972/128824584-a074a8aa-f508-4795-a4b6-66119c22b895.gif)

**Comment by markotoplak** [comment]
Works properly for me (Linux, Qt 5.15.4). Please report your OS and Qt versions (you can also experiment slightly with the latter).

**Comment by markotoplak** [comment]
Interestingly, your screenshot shows that there was one instant when the whole text was selected and then it changed. :)

**Comment by PrimozGodec** [comment]
I can reproduce it on macOS Big Sur with both qt versions: 5.12, 5.15
For me it does not look that we are doing something wrong with listView

**Comment by PrimozGodec** [comment]
It is definitely PyQt - Qt issue. I can reproduce it in a simple application that does not use any orange widget component.
```
from PyQt5.QtWidgets import QApplication, QLineEdit

import sys

app = QApplication(sys.argv)

window = QLineEdit()
window.show()  # IMPORTANT!!!!! Windows are hidden by default.

app.exec()
```

**Comment by janezd** [comment]
We can't do anything then, and can close the issue, right?

**Comment by markotoplak** [comment]
We should find the corresponding Qt bug, if it exists, else file one. If the bug appears in pure Qt, we'd need to file a bug to the Qt, else PyQt.

**Comment by janezd** [comment]
Looks like this? https://bugreports.qt.io/browse/QTBUG-75902

**Comment by ajdapretnar** [comment]
Yes, even though the report says "Linux", but I am experiencing it on OSX. But yes, the same thing.

**Comment by markotoplak** [comment]
Ok, so we can't do much about it, and worse, Qt 5.15 might not ever fix the bug. @ajdapretnar or @PrimozGodec, could you check what happens in PyQt6? So that we know what we expect in the future, or even report a Qt bug.

Just install it PyQt6 (to be safe, perhaps a new environment), run the following code, and check how the selection works.

```
from PyQt6.QtWidgets import *
app = QApplication([])
label = QLineEdit('Hello World!')
label.show()
app.exec()
```
